external/storm-hdfs/src/main/java/org/apache/storm/hdfs/bolt/HdfsBolt.java

```diff
@@ -80,6 +86,11 @@ public HdfsBolt addRotationAction(RotationAction action){
         return this;
     }
 
+    public HdfsBolt withTickTupleIntervalSeconds(int interval) {
```

Comment:
I like that change since it separates the function from the implementation.  A similar change could be made to HiveBolt, to which I also added a tick tuple functionality recently.  If this PR is otherwise ok, I'd like to open a new JIRA to change this bolt and HiveBolt at the same time.